<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>IMProfileWidget</class>
 <widget class="QWidget" name="IMProfileWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>78</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="ScrollFrame" name="accountScrollFrame" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>13</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addIMAccountButton" >
       <property name="toolTip" >
        <string>Add an instant messaging account (MSN, AIM, GoogleTalk...)</string>
       </property>
       <property name="text" >
        <string>Add an IM account</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../../resources/wengophone.qrc" >:/pics/add.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="ToolTipLineEdit" name="aliasLineEdit" >
     <property name="toolTip" >
      <string>Enter your nickname here</string>
     </property>
     <property name="text" >
      <string>Enter your nickname here</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="0" >
    <widget class="QToolButton" name="avatarButton" >
     <property name="toolTip" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Click to change your avatar&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="icon" >
      <iconset resource="../../../../resources/wengophone.qrc" >:/pics/avatar_background.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>ToolTipLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ToolTipLineEditProxy.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>ScrollFrame</class>
   <extends>QFrame</extends>
   <header>qtutil/ScrollFrame.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>avatarButton</tabstop>
  <tabstop>aliasLineEdit</tabstop>
  <tabstop>addIMAccountButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources/wengophone.qrc" />
 </resources>
 <connections/>
</ui>
